sdks/typescript/boot.go

```diff
@@ -169,6 +169,6 @@ func main() {
 		args = append(args, "--status_endpoint="+info.GetStatusEndpoint().GetUrl())
 	}
 
-	log.Fatalf("User program exited: %v", execx.Execute("npx", args...))
+	log.Printf("User program exited: %v", execx.Execute("npx", args...))
 	log.Printf("SDK exited cleanly.")
```

Comment:
Similarly here, we probably should just remove the Printf statement since if it makes it to the user, this is a failure case.
```suggestion
	log.Fatalf("User program exited: %v", execx.Execute("npx", args...))
```